packages/peregrine/lib/context/cart.js

```diff
@@ -72,3 +90,19 @@ export default connect(
 )(CartContextProvider);
 
 export const useCartContext = () => useContext(CartContext);
+
+const CREATE_CART_MUTATION = gql`
```

Comment:
Add comments about the exception to the rule which is that we are including gql in peregrine here.